AzureCalling/Utilities/ThemeColor.swift

```diff
@@ -0,0 +1,13 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import Foundation
+import UIKit
+
+enum ThemeColor {
+    static var primary = UIColor(named: "PrimaryColor") ?? UIColor(red: 0, green: 120 / 255, blue: 212 / 255, alpha: 1)
+
+    static var gray300 = UIColor(named: "Gray300") ?? UIColor(red: 172 / 255, green: 172 / 255, blue: 172 / 255, alpha: 1)
```

Comment:
Let's remove the defaults here. 

In this situation, we are expecting to always have the color in the assets. If we provide a default, it might actually hide the problem if the asset was removed/renamed.

Reply:
Sounds good. Removed